### PR TITLE
feature(runner): change test runner to be in terminal

### PR DESCRIPTION
**Pull request by joshbDev**

### CODE REVIEW

**Comment by dandelany** [comment]
Thanks for this @thehindenburg ! I'm currently making a lot of structural changes in the `0.2.1-combined` branch which will become the new 0.3.0 release. I'll make sure to include this in my changes, will update this ticket when it ships.

File: package.json [hunk 1]
```diff
@@ -23,9 +23,7 @@
     "moment": "^2.10.6",
     "numeral": "^1.5.3",
     "react-addons-shallow-compare": "^0.14.7",
-    "recompose": "^0.15.1"
-  },
-  "peerDependencies": {
+    "recompose": "^0.15.1",
     "react": "^0.14.2",
```
Comment (dandelany) [inline package.json]: `react` and `react-dom` should actually stay in peerDependencies - otherwise Reactochart might install and use its own version of React when included in a project, which is bad and breaks React. Will update this in my batch of changes coming soon

Comment (thehindenburg) [inline package.json]: With the new version of NPM peerDependencies aren't being installed automatically 
https://docs.npmjs.com/files/package.json#peerdependencies

so be aware of that when making your changes. Also a lot of the tests are currently broken, I'm going to submit another PR to fix them

Comment (dandelany) [inline package.json]: ack don't touch tests yet, i'm working on them too right now

Comment (dandelany) [inline package.json]: also yeah i'm aware of that note re: peerDependencies - it's been that way for awhile, but they still need to be peerDependencies. The only way this would cause an issue is if someone tried to install Reactochart in a project that didn't use React (which would be silly) - in which case they would get a warning upon `npm install`

Comment (dandelany) [inline package.json]: @thehindenburg if you want to work on any additional changes this week before the new branch gets released:

* check w/ me first so we don't step on each other's toes
* branch off of the `0.2.1-combined` branch and PR back into that branch.

Thanks!

**Comment by krissalvador27** [comment]
@dandelany @joshbDev is this pull request still relevant?

**Comment by dandelany** [comment]
Nope @krissalvador27  this is an old one and the relevant changes here were included in the 0.3 release. Thanks for checking! Closing.
